mimic/rest/auth_api.py

```diff
@@ -190,6 +191,46 @@ def validate_token(self, request, token_id):
             response["access"]["RAX-AUTH:impersonator"] = impersonator_user_role(
```

Comment:
Non-blocking - just wanted to make sure that it's ok for this logic that sets the impersonation token to get run even if it's one of the tenants in the hard-coded maas roles?